Integrations/Slack/Slack.py

```diff
@@ -826,20 +923,42 @@ def slack_send():
     if not (to or group or channel):
         return_error('Either a user, group or channel must be provided.')
 
-    entitlement_match = re.search(ENTITLEMENT_REGEX, message)
-    if entitlement_match:
-        try:
-            parsed_message = json.loads(message)
-            entitlement = parsed_message['entitlement']
-            message = parsed_message['message']
-        except Exception:
-            pass
+    reply = ''
+    expiry = ''
+    default_response = ''
+    if blocks:
+        entitlement_match = re.search(ENTITLEMENT_REGEX, blocks)
+        if entitlement_match:
+            try:
+                parsed_message = json.loads(blocks)
+                entitlement = parsed_message.get('entitlement')
+                blocks = parsed_message.get('blocks')
+                reply = parsed_message.get('reply')
+                expiry = parsed_message.get('expiry')
+                default_response = parsed_message.get('default_response')
+            except Exception:
+                demisto.info('Slack - could not parse JSON from entitlement blocks.')
```

Comment:
lets also log the value of `blocks`

Reply:
I think it could contain sensitive information...